src/sentry/static/sentry/app/components/sidebar/sidebarDropdown/index.tsx

```diff
@@ -0,0 +1,228 @@
+                  {hasOrganization ? org.name : user.name}{' '}
+                  <i className="icon-arrow-down" />
+                </OrgOrUserName>
+                <UserNameOrEmail>
+                  {hasOrganization ? user.name : user.email}
+                </UserNameOrEmail>
+              </OrgAndUserWrapper>
+            )}
+          </SidebarDropdownActor>
+
+          {isOpen && (
+            <OrgAndUserMenu {...getMenuProps({})}>
+              {hasOrganization && (
+                <React.Fragment>
+                  <SidebarOrgSummary organization={org} />
+                  {hasOrgRead && (
+                    <SidebarMenuItem to={`/settings/${org.slug}/`}>
+                      {t('Organization settings')}
+                    </SidebarMenuItem>
+                  )}
+                  {hasMemberRead && (
+                    <SidebarMenuItem to={`/settings/${org.slug}/members/`}>
+                      {t('Members')}
+                    </SidebarMenuItem>
+                  )}
+
+                  {hasTeamRead && (
+                    <SidebarMenuItem to={`/settings/${org.slug}/teams/`}>
+                      {t('Teams')}
+                    </SidebarMenuItem>
+                  )}
+
+                  <Hook name="sidebar:organization-dropdown-menu" organization={org} />
+
+                  {!config.singleOrganization && (
+                    <SidebarMenuItem>
+                      <SwitchOrganization canCreateOrganization={canCreateOrg} />
+                    </SidebarMenuItem>
+                  )}
+
+                  <Divider />
+                </React.Fragment>
+              )}
+
+              {!!user && (
+                <React.Fragment>
+                  <UserSummary to="/settings/account/details/">
+                    <UserBadgeNoOverflow user={user} avatarSize={32} />
+                  </UserSummary>
+
+                  <div>
+                    <SidebarMenuItem to="/settings/account/">
+                      {t('User settings')}
+                    </SidebarMenuItem>
+                    <SidebarMenuItem to="/settings/account/api/">
+                      {t('API keys')}
+                    </SidebarMenuItem>
+                    {user.isSuperuser && (
+                      <SidebarMenuItem to="/manage/">{t('Admin')}</SidebarMenuItem>
+                    )}
+                    <SidebarMenuItem data-test-id="sidebarSignout" onClick={handleLogout}>
+                      {t('Sign out')}
+                    </SidebarMenuItem>
+                  </div>
+                </React.Fragment>
+              )}
+            </OrgAndUserMenu>
+          )}
+        </SidebarDropdownRoot>
+      )}
+    </DropdownMenu>
+  );
+};
+
+export default withApi(SidebarDropdown);
+
+const SentryLink = styled(Link)`
+  color: ${p => p.theme.white};
+  &:hover {
+    color: ${p => p.theme.white};
+  }
+`;
+
+const UserSummary = styled(Link)`
+  ${menuItemStyles}
+  padding: 10px 15px;
```

Comment:
Shouldn't it be better to use some spaces that we already have  defined in `app/styles/space`?

Reply:
Yeah we should, but this is all existing code. I didn't want to change too much of it.